src/ptvsd/_vendored/pydevd/pydevd.py

```diff
@@ -818,21 +821,36 @@ def apply_files_filter(self, frame, filename, force_check_project_scope):
                 if exclude_by_filter is not None:
                     if exclude_by_filter:
                         # ignore files matching stepping filters
-                        # print('exclude (filtered out): %s' % filename)
+                        if debug_trace_level >= 3:
+                            pydev_log.error_once('File not traced (excluded by filters): %s', filename)
+
                         self._apply_filter_cache[cache_key] = True
                         return True
                     else:
                         # print('include (explicitly included): %s' % filename)
+                        if debug_trace_level >= 3:
```

Comment:
how about removing it?

Reply:
You mean remove the logging?

Reply:
no. I meant the comment above('# print...')

Reply:
Done